diff -cNr schily-2020-11-25-org/RULES/os-msys_nt-10.0-19042.id schily-2020-11-25/RULES/os-msys_nt-10.0-19042.id
*** schily-2020-11-25-org/RULES/os-msys_nt-10.0-19042.id	1970-01-01 09:00:00.000000000 +0900
--- schily-2020-11-25/RULES/os-msys_nt-10.0-19042.id	2020-12-06 19:36:29.092289300 +0900
***************
*** 0 ****
--- 1,25 ----
+ #ident "@(#)os-cygwin_nt-10.0.id	1.1 15/08/03 "
+ ###########################################################################
+ # Written 2015 by J. Schilling
+ ###########################################################################
+ #
+ # OS specific MACRO definitions for Cygnus GCC for MS Windows NT-10.0 (Win 10.0)
+ #
+ ###########################################################################
+ # Copyright (c) J. Schilling
+ ###########################################################################
+ # The contents of this file are subject to the terms of the
+ # Common Development and Distribution License, Version 1.0 only
+ # (the "License").  You may not use this file except in compliance
+ # with the License.
+ #
+ # See the file CDDL.Schily.txt in this distribution for details.
+ # A copy of the CDDL is also available via the Internet at
+ # http://www.opensource.org/licenses/cddl1.txt
+ #
+ # When distributing Covered Code, include this CDDL HEADER in each
+ # file and include the License file CDDL.Schily.txt from this distribution.
+ ###########################################################################
+ include $(SRCROOT)/$(RULESDIR)/os-cygwin32_nt.id
+ 
+ WINNTDEFS +=	-DWINVER=0x0A00 -D_WIN32_WINNT=0x0A00
diff -cNr schily-2020-11-25-org/sccs/sccs/cmd/src/diff.c schily-2020-11-25/sccs/sccs/cmd/src/diff.c
*** schily-2020-11-25-org/sccs/sccs/cmd/src/diff.c	2020-09-19 20:53:33.000000000 +0900
--- schily-2020-11-25/sccs/sccs/cmd/src/diff.c	2020-12-06 19:36:29.095289500 +0900
***************
*** 161,166 ****
--- 161,167 ----
  #include <schily/sysexits.h>
  #define	VMS_VFORK_OK
  #include <schily/vfork.h>
+ #include <schily/io.h>
  #include <schily/libport.h>
  
  #else	/* non-portable SunOS -only definitions BEGIN */
